docs/src/static-pages/website-contribution-guide.mdx

```diff
@@ -68,6 +70,10 @@ Your file must begin with a small block of text that includes some details about
 
 Underneath the page information block, you can now add your markdown content.
 
+Should your new page require a table of contents, after the information block created above just add the following and once the page goes live a table of contents will be automatically created:
```

Comment:
Updating to relfect the guide of if people want to add a ToC they can - we decided to make this optional as some pages if they are just images e.g. photography - they don't need a ToC